src/test/java/HomeWork21.java

```diff
@@ -0,0 +1,16 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class HomeWork21 extends BaseTest{
+
+    @Test
+    public void renamePlaylist () throws InterruptedException {
+        openLoginUrl();
+        enterEmail("beomseo.park@testpro.io");
+        enterPassword("te$t$tudent");
+        clickSubmit();
+        openOption();
+        clickEdit();
+        rename("66");
+    }
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```

Please, add Assert

**One test - One Assert**